spec/support/rails_template.rb

```diff
@@ -91,5 +100,10 @@
   rails_command "parallel:drop parallel:create parallel:load_schema", env: ENV['RAILS_ENV']
 end
 
+# Require turbolinks if necessary
+if ENV["BUNDLE_GEMFILE"] == File.expand_path("../../gemfiles/rails_60_turbolinks/Gemfile", __dir__)
+  append_file 'app/assets/javascripts/active_admin.js', "//= require turbolinks\n"
+end
+
```

Comment:
This is super picky, but nothing changed regarding `turbolinks`and still these lines have been shifted. Can we move them back to where they were to reduce and simplify the final diff?